source/js/components/accordion/accordion.js

```diff
@@ -16,7 +16,7 @@ class Accordion {
   bindEvents() {
     this.title.addEventListener("click", (e) => {
       e.preventDefault();
-      let open = !this.content.classList.contains("tw-hidden");
+      let open = this.title.getAttribute("aria-expanded") === "true";
```

Comment:
Checking `aria-expanded` instead as mobile nav doesn't use `tw-hidden` to toggle visibility.